### ISSUE TITLE
Please add multiple number account support in single molly app.

### ISSUE THREAD

**Issue by danialjose** [mollyim/mollyim-android#97]
### Is there an existing request for this?

- [X] I have searched the existing issues

### Feature description

I have a dual sim phone and one virtual phone number, so i would like to see an option to add multiple accounts in molly. For example there is  client for telegram named nekogram x wich support multiple telegram accounts,

**Comment by HyperOrnstein** [comment]
+1 on my side! Would really be great!
If it's hard to do, maybe you can change the App-ID of Molly or Molly-FOSS, so we can install them both on the same device/profile?
Thank you for your awesome work!

**Comment by Sandyran** [comment]
Yeah, I would love this kind of functionality. I have two phone numbers that I use for different purposes, and being able to use two Signal accounts at the same time, would be nice.
